### ISSUE TITLE
WARN found no layout file for "HTML" for kind "home"

### ISSUE THREAD

**Issue by adam-sroka** [jpanther/congo#382]
### What happened?

Hi there, and thanks for the awesome theme 😊!

Yesterday, when I was about to deploy my site (on [Codeberg pages](https://codeberg.page), by pushing hugo output to a `pages` branch) the build step failed (both with hugo `v0.98.0` and also after updating to `v0.106.0`), generating this output:

```
➜ website git:(main) ✗ hugo serve
Start building sites …
hugo v0.98.0+extended darwin/amd64 BuildDate=unknown
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "JSON" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

                   | EN
-------------------+-----
  Pages            | 13
  Paginator pages  |  0
  Non-page files   | 10
  Static files     | 11
  Processed images |  0
  Aliases          |  0
  Sitemaps         |  1
  Cleaned          |  0

Built in 30 ms
```
(output for `hugo serve` here, but same WARN messages were happening for `hugo`, `hugo --minify`, etc.)

I spent quite some time debugging and trying to find out what's wrong, because I've been using congo without issues before, but nothing seemed to work. Only when I started a new fresh hugo site with congo and got the same error did I realise that this probably isn't related to some config change that I did to my site.

I then remembered that hugo re-fetched the congo mod when I ran the initial `hugo` build command after which all of this started happening, and then I found out about the [`hugo mod clean`command](https://gohugo.io/commands/hugo_mod_clean/) command to clear the module cache, which actually solved this problem.

Since I tried searching this error in this repo's issues and wiki and couldn't find anything (sorry if I just missed it!) I'm posting this issue (and closing it after), so that anyone who encounters something like this in the future can hopefully save a bit of time.

tl;dr just run:

```bash
hugo mod clean
```

and then

```bash
hugo mod get -u
```

Cheers 🙌!

### Theme version

v2.4.1

### Hugo version

v0.98.0

### Relevant Hugo log output

```shell
➜ website git:(main) ✗ hugo serve
Start building sites …
hugo v0.98.0+extended darwin/amd64 BuildDate=unknown
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "section": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "taxonomy": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "term": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "HTML" for kind "page": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.
WARN 2022/11/21 20:53:16 found no layout file for "JSON" for kind "home": You should create a template file which matches Hugo Layouts Lookup Rules for this combination.

                   | EN
-------------------+-----
  Pages            | 13
  Paginator pages  |  0
  Non-page files   | 10
  Static files     | 11
  Processed images |  0
  Aliases          |  0
  Sitemaps         |  1
  Cleaned          |  0

Built in 30 ms
```

**Comment by jpanther** [comment]
Thanks for the detailed report, @adam-sroka. I think that this is one of those error messages that pops up for multiple reasons and is not always an obvious solution. I know others have reported similar but it seems like they were a different issue - #50, #295. In any case, this is very useful information.

I might look to add something about the `clean` command into the docs because it's not one that a lot of people seem to know about.

**Comment by adam-sroka** [comment]
Thanks a lot @jpanther! Yup, this error message seems to pop up for a variety of reasons, so linking this issue might be nice for those that are looking up the error message in this repos issues/wiki, so that they might also try the `clean` command, since it can perhaps help 😊.

**Comment by adam-sroka** [comment]
Updated the issue name with the "HTML" in the error message as that seems to be the more common one.
